app/controllers/api/v1/shelters_controller.rb

```diff
@@ -5,43 +5,56 @@ class Api::V1::SheltersController < ApplicationController
   end
 
   def index
-    @filters = {}
-    @shelters = Shelter.all
+    @shelters, @filters = apply_params(Shelter.all, params)
+  end
+
+  def outdated
+    @outdated, @filters = apply_params(
+      Shelter.where("updated_at < ?", 4.hours.ago).order('updated_at DESC'),
```

Comment:
Just some food for thought (not a review).  This query is a good candidate for a scope `Shelter.outdated`.  The definition would be placed in the Shelter model after the after_commit callback:
```Ruby
scope :outdated, ->(timing = 4.hours.ago) { where("updated_at < ?", timing) }
```

Reply:
Nice, hadn't used those before. I'll take a look at that.